tst/com/amazon/corretto/crypto/provider/test/HmacTest.java

```diff
@@ -66,10 +74,75 @@ public class HmacTest {
     }
   }
 
+  // TODO: This version needs to be updated once known.
+  private static final String MINIMUM_VERSION_WITH_PRECOMPUTED_KEY = "2.4.1";
+
+  /**
+   * Call this function before any tests that use precomputed keys.
+   *
+   * <p>TODO: This function will need to be changed once FIPS support precomputed keys.
+   */
+  private void assumePrecomputedKeySupport() {
+    TestUtil.assumeMinimumVersion(
+        MINIMUM_VERSION_WITH_PRECOMPUTED_KEY, AmazonCorrettoCryptoProvider.INSTANCE);
```

Comment:
do we need this check? because this PR's contents will be merged as one unit, these tests will never be present without support for WithPrecomputedKey.